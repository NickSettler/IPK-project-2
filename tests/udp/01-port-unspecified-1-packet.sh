./ipk-sniffer -i en0 --udp