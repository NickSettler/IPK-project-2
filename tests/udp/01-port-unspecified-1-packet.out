timestamp: 2023-04-17T14:16:52.859+02:00
src MAC: b0:be:83:4a:c4:b0
dst MAC: 98:77:e7:50:60:5e
frame length: 105 bytes
src IP: 2a00:1028:83a2:25ea:cc68:af55:570a:888b
dst IP: 2a00:1028:83a2:25ea:9a77:e7ff:fe50:605e
src port: 52906
dst port: 53

0x0000: 98 77 e7 50 60 5e b0 be 83 4a c4 b0 86 dd 60 0d  .w.P`^...J....`.
0x0010: 0d 00 00 33 11 40 2a 00 10 28 83 a2 25 ea cc 68  ...3.@*..(..%..h
0x0020: af 55 57 0a 88 8b 2a 00 10 28 83 a2 25 ea 9a 77  .UW...*..(..%..w
0x0030: e7 ff fe 50 60 5e ce aa 00 35 00 33 9c 56 50 fa  ...P`^...5.3.VP.
0x0040: 01 00 00 01 00 00 00 00 00 00 0b 63 6c 6f 75 64  ...........cloud
0x0050: 63 6f 6e 66 69 67 09 6a 65 74 62 72 61 69 6e 73  config.jetbrains
0x0060: 03 63 6f 6d 00 00 01 00 01                       .com.....

